README.md

```diff
@@ -32,5 +33,8 @@ See the [contributing guide](CONTRIBUTING.md) for detailed instructions on how t
 
 ### Knowledge Management
 
+### Mind Apps
```

Comment:
I don't think this is a sub category, I think this app is a knowledge management app so just remove this.